.github/workflows/deploy-to-eks.yaml

```
@@ -12,6 +12,9 @@ on:
       uats_branch:
         description: 'Branch to run the UATs from e.g. main or track/1.9. By default, this is defined by the dependencies.yaml file.'
         required: false
+      bundle_url:
+        description: 'URL to be used for version checking. It is generally (starting from 1.10 release) hardcoded in the UATs, so do not specify unless you want to perform a custom check.'
```

Won't it be hardcoded in 1.9 too? In any case, I 'd rephrase it for a bit of more clarity
```suggestion
        description: 'URL to be used for version checking. Starting from 1.10 release, it is hardcoded in the UATs by default, so leave this empty unless you want to perform a custom check.'
```